core/src/main/java/org/apache/accumulo/core/conf/ClientProperty.java

```diff
@@ -61,6 +61,14 @@
       "Change the" + " durability for the BatchWriter session. To use the table's durability"
           + " setting. use \"default\" which is the table's durability setting."),
 
+  // Scanner
+  SCANNER_BATCH_SIZE("scanner.batch.size", "1000",
```

Comment:
This should also be used in the factory methods for creating a scanner and batch scanner, setting the default for those.

Reply:
It is used in the factory method for scanner.

Reply:
ok I see.  When I looked at the diffs I thought it was setting it only for the batch deleter